terra-jupyter-base/custom/edit-mode.js

```diff
@@ -186,14 +186,16 @@ define(() => {
         }
     }
 
-    function handleMetaSuccess(res) {
+    function handleMetaSuccess(res, shouldStaggerCalls) {
         toggleMetaFailureBanner(false); // sets banner for meta status
 
         const isEditMode = res.syncMode == 'EDIT';
 
         if (isEditMode) {
             handleCheckMetaResp(res); // displays modal if theres an issue in the payload
-            getLock(res); // gets lock if in edit mode
+            //if its not the first time we are acquiring lock, wait half the time until next check meta call before calling this.
+            waitTime = shouldStaggerCalls ? lastLockedTimer / 2 : 0
```

Comment:
Currently, checkMeta occurs every 60 seconds and autosave occurs every 5 seconds. the /2.2 ensures we stagger by approximately 27 seconds, which should be off-cycle to both.